modules/system/defaults/controlcenter.nix

```
@@ -1,100 +1,397 @@
 { config, lib, ... }:
-
+let
+  mkEnumApply =
+    mapping: default: v:
+    if v == null then
+      null
+    else if mapping ? v then
+      mapping.${v}
+    else
+      default;
+  mkBoolApply =
+    mapping: v:
+    if v == null then
+      null
+    else if v then
+      mapping.true
+    else
+      mapping.false;
+in
 {
   options = {
 
-    system.defaults.controlcenter.BatteryShowPercentage = lib.mkOption {
-        type = lib.types.nullOr lib.types.bool;
-        default = null; 
-        description = ''
-            Apple menu > System Preferences > Control Center > Battery
+    system.defaults.controlcenter.AccessibilityShortcuts = lib.mkOption {
+      type = lib.types.nullOr (
+        lib.types.enum [
+          "both"
+          "menuBar"
+          "controlCenter"
+          "hide"
+        ]
+      );
+      default = null;
+      apply = mkEnumApply {
+        both = 3;
+        menuBar = 6;
+        controlCenter = 9;
+        hide = 12;
+      } 12;
+      description = ''
+        Apple menu > System Preferences > Control Center > Accessibility Shortcuts
 
-            Show a battery percentage in menu bar. Default is null.
-        '';
+        Options:
+          both           - Show in Menu Bar and Control Center (3)
+          menuBar        - Show in Menu Bar only (6)
+          controlCenter  - Show in Control Center only (1 or 9)
+          hide           - Don't Show (2, 4, 8, or 12)
+      '';
     };
 
-    system.defaults.controlcenter.Sound = lib.mkOption {
-        type = lib.types.nullOr lib.types.bool;
-        apply = v: if v == null then null else if v == true then 18 else 24;
-        default = null; 
-        description = ''
-            Apple menu > System Preferences > Control Center > Sound
+    system.defaults.controlcenter.AirDrop = lib.mkOption {
+      type = lib.types.nullOr lib.types.bool;
+      default = null;
+      apply = mkBoolApply {
+        true = 18;
+        false = 24;
+      };
+      description = ''
+        Apple menu > System Preferences > Control Center > AirDrop
 
-            Show a sound control in menu bar . Default is null.
+        Show a AirDrop control in menu bar. Default is null.
 
-            18 = Display icon in menu bar
-            24 = Hide icon in menu bar
-        '';
+        18 = Show in Menu Bar
+        24 = Don't Show in Menu Bar
```

```suggestion
        true = Show in Menu Bar
        false = Don't Show in Menu Bar
```
Or remove this section of the description?